fineract-savings/src/main/java/org/apache/fineract/portfolio/savings/service/GSIMReadPlatformServiceImpl.java

```diff
@@ -73,8 +73,11 @@ public GroupSavingsIndividualMonitoringAccountData mapRow(final ResultSet rs, @S
             final BigDecimal parentBalance = rs.getBigDecimal("parentBalance");
             final BigDecimal childBalance = rs.getBigDecimal("childBalance");
             final String savingsStatus = SavingsAccountStatusType.fromInt((int) rs.getLong("savingsStatus")).toString();
-            return GroupSavingsIndividualMonitoringAccountData.getInstance2(gsimId, groupId, clientId, accountNumber, childAccountId,
-                    childAccountNumber, parentBalance, childBalance, childAccountsCount, savingsStatus);
+
+            return GroupSavingsIndividualMonitoringAccountData.builder().gsimId(gsimId).groupId(groupId).clientId(clientId)
+                    .accountNumber(accountNumber).childAccountId(childAccountId).childAccountNumber(childAccountNumber)
+                    .childDeposit(parentBalance).parentDeposit(childBalance).childAccountsCount(childAccountsCount)
```

Comment:
When assigning `parentBalance` to `childDeposite` and assign `childBalance` to `parentDeposit`, it's a bit ambiguous to me due to convention, but I just followed the previous implementation.

Reply:
if you think it should be modified please let me know.

Reply:
@rrpawar96 Thoughts?